components/library/libjpeg6-ijg/pkg5

```diff
@@ -1,11 +0,0 @@
-{
-    "dependencies": [
-        "SUNWcs",
-        "system/library"
-    ],
-    "fmris": [
-        "image/library/libjpeg6",
-        "image/library/libjpeg6-ijg"
```

Comment:
These packages are required by other packages, so you cannot simply remove them.

Reply:
I can also rename them to the new libjpeg package. The project to replace those two packages started 5 years ago. If they still used I consider that a Bug that needs fixing.

Reply:
If you want to remove `components/library/libjpeg6-ijg` then you need either obsolete or rename both `image/library/libjpeg6` and `image/library/libjpeg6-ijg`.  Since both packages are runtime and build dependencies of many other packages (for example `desktop/office/abiword`, `desktop/remote-desktop/freerdp`, `runtime/java/openjdk22`, `library/perl-5/tk-538`, `library/qt6`, etc.) then obsoletion does not look correct.  But since they are still massively used I think it would be far better first to make sure they are no longer used, and then remove (obsolete) them.

Reply:
Ok, I'll rename them. As for the dependency. Unfortunately this looks like a severe problems as libjpeg-turbo should have been used. Unfortunately I first must change libjpeg6 to libjpeg9 so libjpeg9 gets detected.